docs/developer/architecture/overview.rst

```diff
@@ -0,0 +1,95 @@
+Overview
+========
+
+High-level architecture
+-----------------------
+
+The diagram below describes the high-level architecture of WebMon, including some external resources
```

Comment:
High-level architecture
Paragraph 1: 
•	I would change the text to indicate the relationship(s) between the different services instead of saying it is message flow.  IMO, this diagram shows structure.  Two objects have to have a relationship before then can communicate.  A suggestion would to add statements like, “ONCat is the experiment data catalog which the autoerducer has a relationship with because this is where the autoreducers catalog’s runs…”